src/style.css

```diff
@@ -2,0 +1,61 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.body {
+  margin: 0;
+}
```

Comment:
You're using the class `.body` to style the body element, which is not a good practice. You should use tag selectors to style the `html` and `body` elements directly. This is one of the few exceptions where it's okay to use tag selectors for styling.